machines/vic/reification/appliance.go

```diff
@@ -0,0 +1,416 @@
+// Copyright 2016 VMware, Inc. All Rights Reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//    http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package reification
```

Comment:
I think this should be named something different since `reification` isn't very enlightening. After talking to Emma, though we're both at a loss as to what it should be called (naming is hard) any suggestions from others? Can package names be two words?
As I understand it this package takes a generated config and translates/uses/applies it for our backend so that users will always generate a consistent config and we can handle different backend implementations through this package.